server.js

```diff
@@ -1,39 +1,167 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-import mongoose from 'mongoose'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
-mongoose.Promise = Promise
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+const express = require("express");
+const cors = require("cors");
+const mongoose = require("mongoose");
+const bodyParser = require('body-parser')
+const { newTopSongs } = require("./data/cleanSongs")
+
+const mongoUrl = "mongodb+srv://new_user90:nMepfddUO7kJmTER@cluster0.d0pe1.mongodb.net/myFirstDatabase?retryWrites=true&w=majority";
```

Comment:
I think this password url should be hidden in the .env file because now it's public to anyone. There was good video in the #help channel by Carol for this.